test/component/CustomsDeclarationStatusSpec.scala

```diff
@@ -99,7 +102,7 @@ class CustomsDeclarationStatusSpec extends ComponentTestSpec with AuditService w
       status(result) shouldBe OK
 
       And("the response body is a valid status xml")
-      contentAsString(result) shouldBe validResponse
+      contentAsString(result) shouldBe validResponse(acceptanceDateVal.toString(DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss'Z'")))
```

Comment:
can date pattern be replaced with `ISODateTimeFormat`

Reply:
ok